include/boost/redis/connection.hpp

```diff
@@ -1250,8 +1251,8 @@ class connection {
       { return impl_.get_executor(); }
 
    /// Calls `boost::redis::basic_connection::async_run`.
-   template <class CompletionToken>
-   auto async_run(config const& cfg, logger l, CompletionToken token)
+   template <class CompletionToken = asio::deferred_t>
+   auto async_run(config const& cfg, logger l, CompletionToken token = {})
```

Comment:
As minor comments, consider unifying all signatures using completion tokens to `CompletionToken&&`